client/my-sites/stats/stats-period-header/style.scss

```diff
@@ -0,0 +1,49 @@
+.stats__period-header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	background-color: var(--color-surface);
+	padding: 16px 0;
+	margin-top: 48px;
+
+	.stats-period-navigation {
+		flex: 1;
+	}
+
+	.segmented-control {
+		margin-left: 24px;
+
+		&.is-primary {
+			.segmented-control__item.is-selected {
+				.segmented-control__link {
+					border-color: var(--color-primary);
+					background-color: var(--color-primary);
+
+					&:hover {
+						background-color: var(--color-primary);
+					}
+				}
+			}
+		}
+
+		.segmented-control__item {
+			&:first-of-type .segmented-control__link {
+				border-top-left-radius: 4px;
```

Comment:
What do you think about using one variable for values like those `4px`? It could be easier to update one thing in the future.

Reply:
I have tried it before, but I decided to leave them there. However, I agree with extracting them when there will be more common usage. I think the use of `border-radius` that not a kind of module and frequently changed property, which keeps the value, is more intuitive than variables.

Reply:
We can revisit this if it becomes an issue in the future 👍